src/commands/emasser/put/controls.ts

```diff
@@ -7,18 +7,58 @@ import {Command, Flags} from '@oclif/core'
 import {outputError} from '../../../utils/emasser/outputError'
 import {ApiConnection} from '../../../utils/emasser/apiConnection'
 import {outputFormat} from '../../../utils/emasser/outputFormatter'
-import {FlagOptions, getFlagsForEndpoint, getJsonExamples} from '../../../utils/emasser/utilities'
+import {FlagOptions, getFlagsForEndpoint, getJsonExamples, printHelpMsg, printRedMsg} from '../../../utils/emasser/utilities'
 
 import {ControlsApi} from '@mitre/emass_client'
-import {ControlsResponsePut,
-  ControlsGet as Controls} from '@mitre/emass_client/dist/api'
-
-function printHelpMsg() {
-  console.log('\x1B[93m', '\nInvoke saf emasser put controls [-h, --help] for additional help', '\x1B[0m')
+// import {ControlsResponsePut,
```

Comment:
Should this line be deleted?

Reply:
Because there are comments without a return statment.

Reply:
nope not needed